README.md

```diff
@@ -362,3 +365,8 @@ See [CONTRIBUTING.md](https://github.com/pantheon-systems/pantheon-advanced-page
 
 ### 0.1.0 (November 23rd, 2016) ###
 * Initial release.
+
+## Upgrade Notice ##
+
+### Latest ###
+**Note that the Pantheon Advanced Page Cache Latest release now prefixes keys on WPMS with the blog ID. For users who already have this plugin installed on a WPMS, they will need to click the Clear Cache button on the settings page to generate the prefixed keys.
```

Comment:
🔢 
```suggestion
Note that the Pantheon Advanced Page Cache Latest release now prefixes keys on a WordPress Multisite (WPMS) with the blog ID. For users who already have this plugin installed on a WPMS, they will need to click the Clear Cache button on the settings page to generate the prefixed keys.
```

Also, are you going for bullets here with ** or is that meant to be read as an asterisk? I would drop either way but optional.